Dockerfile

```diff
@@ -1,15 +1,20 @@
-FROM golang:1.11.5
+FROM golang:1.12
```

Comment:
Maybe not move docker image to 1.12 yet (at least in this PR). Since we may do a 0.12.2 soon, we may want to stay at same go release of 1.11.5 compared to streaming 0.12.0.